src/kid.js

```diff
@@ -1,6 +1,14 @@
 import { mod10_generate, mod10_validate } from './mod10'
 import { mod11_generate, mod11_validate } from './mod11'
 
+/**
+ * 
+ * @param {Object} params
+ * @param {number} params.length Length of the kidnumber
+ * @param {String[]} params.baseNum The basenumbers
+ * @param {String} params.modulo The modulo
```

Comment:
`{String} modulo The modulo` is ambigous. See other comment

Reply:
fixed by f9aa173